zcash_client_sqlite/Cargo.toml

```diff
@@ -13,19 +13,24 @@ license = "MIT OR Apache-2.0"
 edition = "2018"
 
 [dependencies]
+async-trait = "0.1.52"
 bech32 = "0.9.1"
 bs58 = { version = "0.4", features = ["check"] }
 ff = "0.8"
+futures = { version = "0.3", package = "futures", features = ["compat", "async-await"] }
 group = "0.8"
 jubjub = "0.5.1"
 protobuf = "2.20"
 rand_core = "0.5.1"
-rusqlite = { version = "0.28", features = ["time"] }
+rusqlite = { version = "0.28", features = ["bundled", "time"] }
 libsqlite3-sys= { version = "0.25.2", features = ["bundled"] }
 time = "0.3"
+tokio = { version = "1.20", features = ["full"] }
```

Comment:
Q: Are we really using all the tokio features?

Reply:
same as above. fixed